server/routes/grafana/grafana_routes.py

```
@@ -115,20 +116,20 @@ def alert_webhook(user_id: str):
     if not row_exists or (row_exists and not is_active):
         skip_rca = True
         if not row_exists:
-            logger.info("[GRAFANA] Auto-connecting user %s via webhook", user_id)
+            logger.info("[GRAFANA] Auto-connecting user %s via webhook", sanitize(user_id))
             try:
                 store_tokens_in_db(user_id, {}, "grafana")
             except Exception as exc:
-                logger.exception("[GRAFANA] Failed to auto-connect user %s: %s", user_id, exc)
+                logger.exception("[GRAFANA] Failed to auto-connect user %s: %s", sanitize(user_id), exc)
```

Ruff (0.15.11)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)